.github/workflows/check-for-new-cascadia.yml

```diff
@@ -6,25 +6,45 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     steps:
-    - uses: actions/checkout@v2        
+    - uses: octokit/request-action@v2.x
+      id: get_latest_release
```

Comment:
Is there a reason you're using an id here?

Reply:
The output of this step is needed to get the latest tag on the next step (`fromJson(steps.get_latest_release.outputs.data).tag_name`).

Reply:
Of course, my bad